### PR TITLE
logs: Add total response time

### PR DESCRIPTION
**Pull request by petitcl**
By default, morgan only logs the time it took to start writing the response. In some cases, the actual writing of the response takes significantly more time than start writing it. This PR adds a new field in the json logs *only* that represents this time, for monitoring / debugging capabilities.

### CODE REVIEW

File: src/start-express-app.ts [hunk 1]
```diff
@@ -30,20 +30,40 @@ export default async function(options: N9NodeRouting.Options): Promise<N9NodeRou
 		controllers: [options.path + '/**/*.controller.*s'],
 	};
 
+	// Configure morgan
+	morgan.token('total-response-time', (req: Request, res: Response, digits: number): string => {
+		if (!(req as any)._startAt) {
```
Comment (BenjD90) [inline src/start-express-app.ts:35]: @petitcl I suppose that `_startAt` is set by Morgan ?

File: src/start-express-app.ts [hunk 2]
```diff
@@ -30,20 +30,40 @@ export default async function(options: N9NodeRouting.Options): Promise<N9NodeRou
...
+			return;
+		}
+
+		// // calculate diff
+		const reqStartTime = (req as any)._startAt;
+		const resEndTime = process.hrtime();
+		const ms = (resEndTime[0] - reqStartTime[0]) * 1e3 + (resEndTime[1] - reqStartTime[1]) * 1e-6;
+
+		// return truncated value
+		return ms.toFixed(digits === undefined ? 3 : digits);
```
Comment (BenjD90) [inline src/start-express-app.ts]: @petitcl Please use typescript default value syntax

**Review by BenjD90 — CHANGES_REQUESTED** [review]
1 small change, and a possible comment to add.
Could you add a test about this feature too ?

**Comment by petitcl** [comment]
The unit test are not up to date; they fail even on master. They fail because they are expecting a string output in the logs, but a json is outputted instead.
Ok for you if I change the tests to assert a json output ?

File: src/start-express-app.ts (hunk 1 above)
Comment (petitcl) [reply src/start-express-app.ts:35]: yep

**Review by BenjD90 — APPROVED** [review]

**Pull request merged**